### DISCUSSION TITLE
Python example for dbus msg receiving fails

### DISCUSSION THREAD

**Discussion by dnns92** [AsamK/signal-cli#463] [Q&A]
Hey, great work!
just a simple question:  The signal-cli works fine for me. Also I can send and receive messages over the dbus, which works perfectly.

However, the python example provided in the wiki fails, and provides no  meaningful error message. Maybe someone here can help?

My System:
Raspberry pi 4
Raspian OS
Python 3.8

I used signal-cli -u my-user daemon to start the service and then started the python script from an other shell.

This is the error traceback:
```
python example.py
Traceback (most recent call last):
  File "example.py", line 11, in <module>
    remote_object = bus.get(
  File "/home/pi/.local/lib/python3.8/site-packages/pydbus/proxy.py", line 44, in get
    ret = self.con.call_sync(
gi.repository.GLib.Error: g-io-error-quark: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit dbus-org.asamk.Signal.service not found. (36)
```
It looks like it cant connect ot the signal service. This is the example I used is this:

```
def msgRcv (timestamp, source, groupID, message, attachments):
   print ("Message", message, "received in group", signal.getGroupName (groupID))
   return

from pydbus import SystemBus
from gi.repository import GLib

bus = SystemBus()
loop = GLib.MainLoop()


signal = bus.get('org.asamk.Signal')

signal.onMessageReceived = msgRcv
loop.run()
```

**Comment by AsamK** [comment]
there are two dbus buses on each system, a session/user bus and a system bus.
The signal-cli `daemon` command connects to the session bus by default, but your python script connects to the system bus: `bus = SystemBus()`
Feel free to update the wiki, if the example is not clear enough :)
